pkgs/applications/science/biology/slicer3d/default.nix

```diff
@@ -0,0 +1,133 @@
+{
+  stdenv,
+  lib,
+  autoPatchelfHook,
+  makeDesktopItem,
+  glib,
+  fetchzip,
+  pulseaudio,
+  xorg,
+  zlib,
+  fontconfig,
+  libxkbcommon,
+  libGL,
+  libGLU,
+  freetype,
+  cups,
+  unixODBC,
+  libxcrypt-legacy,
+  alsa-lib,
+  postgresql,
+  nss,
+  nspr,
+  hwloc,
+  fetchurl,
+}:
+
+let
+  pname = "Slicer3D";
+  version = "5.6.2";
+
+  hwloc_old = hwloc.overrideAttrs (
+    old:
+    let
+      hwloc_version = "1.11.9";
+    in
+    {
+      version = hwloc_version;
+      src = fetchurl {
+        url = "https://www.open-mpi.org/software/hwloc/v${lib.versions.majorMinor hwloc_version}/downloads/hwloc-${hwloc_version}.tar.bz2";
+        sha256 = "sha256-OUMzGEJI1jyycIqXblfwUzfQO7UMM6owl/9cWnSoUWQ=";
+      };
+    }
+  );
+
+  desktopItem = makeDesktopItem {
+    name = "Slicer3D";
+    exec = "SlicerApp-real";
```

Comment:
If the `meta.mainProgram` is `Slice3D`, then why is `exec` something else?

Reply:
There's another executable called `Slicer` that isn't in `$out/bin` but just `$out`, which is what I meant to put there.
Not sure why I put Slicer3D though. I changed `desktopItem` to point to the `Slicer` executable and I'll fix those in the next push